app/config.go

```diff
@@ -357,6 +357,11 @@ func (a *App) ClientConfigWithComputed() map[string]string {
 	if installationDate, err := a.ch.srv.getSystemInstallDate(); err == nil {
 		respCfg["InstallationDate"] = strconv.FormatInt(installationDate, 10)
 	}
+	if ver, err := a.ch.srv.Store.GetDBSchemaVersion(); err != nil {
+		mlog.Error("Could not get the schema version", mlog.Err(err))
+	} else {
+		respCfg["Version"] = strconv.Itoa(ver)
```

Comment:
Is this the value that shows up in the modal? I am wondering how was the webapp showing the previous value? I don't see any code edited here. :thinking:

Reply:
It was being set by the `model.CurrentVersion` constant.

Reply:
Can we remove that line then?